index.js

```diff
@@ -48,11 +59,12 @@ function runReadmeGenerator(options) {
     }
     const configPath = options.config ? options.config : `${__dirname}/config.json`;
     const config = JSON.parse(fs.readFileSync(configPath));
-    const parametersList = getParameters(options);
+    const parsedMetadataComments = getParsingResults(options);
+    const parametersList = getParameters(options.values, parsedMetadataComments.parsedValues);
 
     if (readmeFilePath) {
       const paramsToRender = buildParamsToRenderList(parametersList, config);
-      const sections = buildSectionsArrays(paramsToRender);
+      const sections = buildSectionsObjects(paramsToRender, parsedMetadataComments.parsedSectionDescriptions);
```

Comment:
If we apply what I described above about the `section` class, this would result in more consistency. We would just need to change the `buildParamsToRedenerList` code